File: Niveau 1/3 – Calculs et découverte des variables/Course avec les enfants.py
from robot import *

x = 1
for i in range(10):
   for j in range(x):
      if x > 10:
         break
      droite()
   ramasser()
   for j in range(x):
      if x > 10:
         break
      gauche()
   x += 1
   deposer()

File: Pascal/Niveau 1/3 – Calculs et découverte des variables/Course avec les enfants.pas
program Solution;
uses Robot;
var
   i, j, ring: LongInt;
begin
   ring := 1;
   for i := 1 to 10 do
   begin
      for j := 1 to ring do
      begin
         droite;
      end;
      ramasser;
      for j := 1 to ring do
      begin
         gauche;
      end;
      deposer;
      ring := ring + 1
   end;
end.